decode.rb

```diff
@@ -0,0 +1,34 @@
+def decode_char(str)
+  decoder = {
+    '.-' => 'A', '-...' => 'B',
+    '-.-.' => 'C', '-..' => 'D',
+    '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V',
+    '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+
+  decoder[str]
+end
+
+def decode_str(str)
+  words_arr = str.split('  ')
+  phrase = ''
```

Comment:
- [ ] Kindly create a ruby method that takes a **_morse code input word_** delimited by a **_single space_** as a parameter. You should split through input word as the next step. The method should iterate through the morse code input word and use the method `decode_char` to translate every letter. 👍 

#### Hint: `decode_word('-- -.--')` should return `"MY"`

![image](https://user-images.githubusercontent.com/77106962/178592641-3042a668-2a4e-4a4e-904b-f013f5e26bd8.png)